kubernetes-deployment/src/chart/Chart.yaml

```
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: <md .Name md>
+name: { { name } }
```

```suggestion
name: {{ name }}
```

Dunno about go, but the elixir liquid lib will choke on that space.

---

it will VS code is being an ass.